akkeeper/src/main/scala/akkeeper/common/Controllers.scala

```diff
@@ -13,23 +13,27 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package akkeeper.master.route
+package akkeeper.common
```

Comment:
The file name `Controllers.scala` is way too generic and assumes that any number of controllers can be located here. Instead can we just name it `Controller.scala` after the name of the base class. Let's also move it to the `akkeeper.common.controller` rather than keeping it in the root `common` package.